### PR TITLE
[doc]fix build and contribute chapter doc errors

### PR DESCRIPTION
**Pull request by Amber1990Zhang**

### CODE REVIEW

File: docs/manual-CN/3.build-develop-and-administration/1.build/1.build-source-code.md [hunk 1]
```diff
@@ -12,43 +15,45 @@ Nebula 基于 C++ 14 开发，因此它需要一个支持 C++ 14 的编译器。
...
+```bash
 bash> git clone https://github.com/vesoft-inc/nebula.git
 ```
 
 #### 步骤 2 : 安装依赖
 
 - 中国用户
 
-    ```
+    ```bash
```
Comment (whitewum) [inline docs/manual-CN/3.build-develop-and-administration/1.build/1.build-source-code.md]: 这里改了，你等laura的pr进去merge一下

File: docs/manual-CN/3.build-develop-and-administration/1.build/1.build-source-code.md [hunk 2]
```diff
@@ -77,17 +82,19 @@ bash> make
 
 - 如果没有任何错误信息
 
-    ```
+    ```sh
```
Comment (laura-ding) [inline docs/manual-CN/3.build-develop-and-administration/1.build/1.build-source-code.md]: It is not sh, it just an output message

File: docs/manual-CN/3.build-develop-and-administration/1.build/1.build-source-code.md [hunk 3]
```diff
@@ -31,44 +35,45 @@ bash> git clone https://github.com/vesoft-inc/nebula.git
...
+  - [CentOS 7.0~7.6](https://nebula-graph.oss-accelerate.aliyuncs.com/third-party/centos7.5.tar.gz)
+  - [CentOS 6.5](https://nebula-graph.oss-accelerate.aliyuncs.com/third-party/centos6.5.tar.gz)
+  - [Ubuntu 1604](https://nebula-graph.oss-accelerate.aliyuncs.com/third-party/ubuntu16.tar.gz)
+  - [Ubuntu 1804](https://nebula-graph.oss-accelerate.aliyuncs.com/third-party/ubuntu18.tar.gz)
 
     步骤 3:
     安装下载好的压缩包
 
-    ```
+    ```sh
```
Comment (laura-ding) [inline docs/manual-CN/3.build-develop-and-administration/1.build/1.build-source-code.md]: United into `bash`

File: docs/manual-CN/3.build-develop-and-administration/2.develop-and-interface/kv-interfaces.md [hunk 4]
```diff
@@ -30,20 +30,20 @@ for (int32_t i = 0; i < 1000; i ++) {
                        std::move(key), std::move(value));
 }
 
-// 通过StorageClient发送请求，相应的参数为spaceId，以及写入的key-value pairs
+// 通过 StorageClient 发送请求，相应的参数为 spaceId，以及写入的 key-value pairs
```
Comment (laura-ding) [inline docs/manual-CN/3.build-develop-and-administration/2.develop-and-interface/kv-interfaces.md]: 通过 StorageClient 发送请求，相应的参数为 spaceId，以及写入的`键值对`

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/deploy-cluster-on-docker.md [hunk 5]
```diff
@@ -178,7 +177,7 @@ Welcome to Nebula Graph (Version 5f656b5)
 
 表明三台集群已部署成功，插入[数据](https://github.com/vesoft-inc/nebula/blob/master/docs/manual-CN/get-started.md#%E5%88%9B%E5%BB%BA%E5%9B%BE%E6%95%B0%E6%8D%AE)进行测试。
 
-```
+```sql
```
Comment (laura-ding) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/deploy-cluster-on-docker.md]: nGQL

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/deploy-cluster-on-docker.md [hunk 6]
```diff
@@ -207,7 +206,7 @@ $ /usr/local/nebula/scripts/nebula.service status storaged
 
 登录 cluster-2，使用 `SHOW HOSTS` 查看
 
-```
+```sql
```
Comment (laura-ding) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/deploy-cluster-on-docker.md]: ditto

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/deploy-cluster-on-docker.md [hunk 7]
```diff
@@ -227,7 +226,7 @@ $ /usr/local/nebula/scripts/nebula.service status storaged
 
 测试数据是否可读
 
-```
+```sql
```
Comment (laura-ding) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/deploy-cluster-on-docker.md]: ditto

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/deploy-cluster.md [hunk 8]
```diff
@@ -13,13 +10,13 @@
 
  `CentOS` 系统：
 
-```
+```sh
```
Comment (laura-ding) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/deploy-cluster.md]: bash

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/storage-metrics.md [hunk 9]
```diff
@@ -49,7 +50,7 @@ _error_qps
 
 根据上面的介绍，就可以写出一个完整的指标名称了，下面是一些示例：
 
-```
+```shell
```
Comment (laura-ding) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/storage-metrics.md]: No shell, They are interfaces

File: docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md [hunk 10]
```diff
@@ -50,7 +50,7 @@ bash> cd nebula && ./build_dep.sh N
 
 3. Install the package
 
-```
+```sh
```
Comment (laura-ding) [inline docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md]: ditto

File: docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/configs-syntax.md [hunk 11]
```diff
@@ -1,20 +1,20 @@
...
-Nebula use `gflags` for run-time configurations.
+Nebula uses `gflags` for run-time configurations.
 
 The related three gflags parameters are: `rocksdb_db_options`, `rocksdb_column_family_options` and `rocksdb_block_based_table_options`.
 
-The three parameters are all in json formet, and the key and value of them are in string format. For example, you can set as follows in the conf file of storage:
+The three parameters are all in json format, and the key and value of them are in string format. For example, you can set as follows in the conf file of storage:
 
-```
+```sh
```
Comment (laura-ding) [inline docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/configs-syntax.md]: it is no shell, they are params for the config file

File: docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/configs-syntax.md [hunk 12]
```diff
@@ -1,20 +1,20 @@
...
     rocksdb_db_options = {"stats_dump_period_sec":"200", "enable_write_thread_adaptive_yield":"false", "write_thread_max_yield_usec":"600"}
     rocksdb_column_family_options = {"max_write_buffer_number":"4", "min_write_buffer_number_to_merge":"2", "max_write_buffer_number_to_maintain":"1"}
     rocksdb_block_based_table_options = {"block_restart_interval":"2"}
 ```
 
 Nebula supports changing some rocksdb parameters in storage service as follows:
 
-```
+```sh
```
Comment (laura-ding) [inline docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/configs-syntax.md]: ditto

File: docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/configs-syntax.md [hunk 13]
```diff
@@ -40,7 +40,10 @@ Nebula supports changing some rocksdb parameters in storage service as follows:
 ```
 
 For example
-`UPDATE VARIABLE storage:rocksdb_column_family_options = { disable_auto_compactions = false , level0_file_num_compaction_trigger = 10 } `
+
+```sql
```
Comment (laura-ding) [inline docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/configs-syntax.md]: nGQL

File: src/tools/spark-sstfile-generator/README.md [hunk 14]
```diff
@@ -70,13 +76,14 @@ Here is an example of mapping file: [mapping.json](mapping.json)
...
 ```
+
 *make sure to keep consistent with DEBUG_LEVEL when building, or there will be some link error like `symbol not found`*
 
 2. run `sbt assembly` to package this project to a spark job jar, whose default name is: `nebula-spark-sstfile-generator.jar`  
 3. run `jar uvf nebula-spark-sstfile-generator.jar librocksdbjni-linux64.so libnebula_native_client.so` to replace the `*.so` files packaged inside the dependency org.rocksdb:rocksdbjni:5.17.2,
     or the following error will occur when spark-submit:
 
-```
+```sh
```
Comment (laura-ding) [inline src/tools/spark-sstfile-generator/README.md]: no sh, it is error info

File: docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/configs-syntax.md [hunk 15]
```diff
@@ -40,7 +40,10 @@ Nebula supports changing some rocksdb parameters in storage service as follows:
...
+UPDATE VARIABLE storage:rocksdb_column_family_options = { disable_auto_compactions = false , level0_file_num_compaction_trigger = 10 }
```
Comment (jievince) [inline docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/configs-syntax.md]: UPDATE CONFIGS

File: docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/storage-balance.md [hunk 16]
```diff
@@ -1,6 +1,6 @@
 # Storage Balance Usage
 
-Nebula's services are composed of  three parts: graphd, storaged and metad. The **balance** in this document focuses on the operation of storage. 
+Nebula's services are composed of  three parts: graphd, storaged and metad. The **balance** in this document focuses on the operation of storage.
```
[the remark this answers is not in the document]
Comment (Amber1990Zhang) [reply docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/storage-balance.md]: good catch

File: docs/manual-CN/3.build-develop-and-administration/1.build/1.build-source-code.md [hunk 17]
```diff
@@ -154,13 +160,13 @@ nebula> SHOW HOSTS;
 
     **步骤 1**: 在 **~/.bashrc** 末添加如下行
 
-    ```
+    ```sh
```
Comment (laura-ding) [inline docs/manual-CN/3.build-develop-and-administration/1.build/1.build-source-code.md]: unified `bash`

File: docs/manual-CN/3.build-develop-and-administration/2.develop-and-interface/kv-interfaces.md [hunk 18]
```diff
@@ -30,20 +30,20 @@ for (int32_t i = 0; i < 1000; i ++) {
                        std::move(key), std::move(value));
 }
 
-// 通过StorageClient发送请求，相应的参数为spaceId，以及写入的key-value pairs
+// 通过 StorageClient 发送请求，相应的参数为 spaceId，以及写入的銉值对
```
Comment (laura-ding) [inline docs/manual-CN/3.build-develop-and-administration/2.develop-and-interface/kv-interfaces.md]: `銉` to `键`

File: docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/deploy-cluster.md [hunk 19]
```diff
@@ -26,15 +26,19 @@ In order to enable multi copy Meta services, you should set the meta addresses s
...
 
-Currently, we support `scripts/services.sh` to manage the nebula cluster.
+Currently, we use `scripts/services.sh` to manage the nebula cluster.
 
-You can `start`, `stop` and `restart` the cluster using this script.
+You can `start`, `stop` and `restart` the cluster with this script.
 
 It looks like the following command:
 
+<<<<<<< HEAD
```
Comment (laura-ding) [inline docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/deploy-cluster.md]: Please resolve the conflict carefully

File: src/tools/spark-sstfile-generator/README.md [hunk 20]
```diff
@@ -56,12 +61,13 @@ usage: nebula spark sst file generator
...
-# Mapping file schema
+### Mapping file schema
 
 The format of the mapping files is json. File Schema is provided as [mapping-schema.json](mapping-schema.json) according to [Json Schema Standard](http://json-schema.org).
 Here is an example of mapping file: [mapping.json](mapping.json)
 
-# FAQ
+### FAQ
+
 ## How to use libnebula-native-client.so under CentOS6.5(2.6.32-431 x86-64)
```
Comment (whitewum) [inline src/tools/spark-sstfile-generator/README.md]: are you sure about the ## or ### ? It looks not logical

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/storage-metrics.md [hunk 21]
```diff
@@ -39,7 +40,7 @@ _error_qps
 
 ### 统计类型
 
-目前支持的类型有 SUM，COUNT，AVG，RATE，以及 P99，P999 等，最高支持到 P999999。
+目前支持的类型有 SUM，COUNT，AVG，RATE，以及 P99，P999 等，最高支持到 P999999。只有 latency 支持 P99 等指标。
```
Comment (whitewum) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/storage-metrics.md]: 我完全没看懂 “只有 latency 支持 P99 等指标。”

File: src/tools/spark-sstfile-generator/README.md (hunk 20 above)
Comment (Amber1990Zhang) [reply src/tools/spark-sstfile-generator/README.md]: There can only be one header with h1 #, changed `## How to use libnebula-native-client.so under CentOS6.5(2.6.32-431 x86-64)` to `#### How to use libnebula-native-client.so under CentOS6.5(2.6.32-431 x86-64)`. This is logical

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/storage-metrics.md (hunk 21 above)
Comment (Amber1990Zhang) [reply docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/storage-metrics.md]: @critical27 please help with this.

File: src/tools/spark-sstfile-generator/README.md [hunk 22]
```diff
@@ -56,27 +61,29 @@ usage: nebula spark sst file generator
...
 
-# Mapping file schema
+### Mapping file schema
 
 The format of the mapping files is json. File Schema is provided as [mapping-schema.json](mapping-schema.json) according to [Json Schema Standard](http://json-schema.org).
 Here is an example of mapping file: [mapping.json](mapping.json)
 
-# FAQ
-## How to use libnebula-native-client.so under CentOS6.5(2.6.32-431 x86-64)
+### FAQ
```
Comment (whitewum) [inline src/tools/spark-sstfile-generator/README.md]: by context, I think FAQ is  H2

Comment (Amber1990Zhang) [reply src/tools/spark-sstfile-generator/README.md]: and so is TODO

**Comment by whitewum** [comment]
remember give some explain about "只有 latency 支持 P99 等指标"

**Comment by Amber1990Zhang** [comment]
> remember give some explain about "只有 latency 支持 P99 等指标"

will fix this here #1224

**Pull request merged**